docs/user/science.rst

```diff
@@ -0,0 +1,142 @@
+      * Familiar with Sphinx? See :doc:`/guides/jupyter`
+
+.. dropdown:: 🔋 Batteries included: Graphs, computations, formulas, maps, diagrams and more
+
+    Take full advantage of getting all the richness of :external+jupyter:doc:`Jupyter Notebook <index>` combined with Sphinx and the giant ecosystem of extensions for both of these.
+
+    Here are some examples:
+
+    * Use symbols familiar from math and physics, build advanced proofs. See also: `sphinx-proof <https://sphinx-proof.readthedocs.io/en/latest/syntax.html>`__
+    * Present results with plots, graphs, images and let users interact directly with your datasets and algorithms. See also: `Matplotlib <https://matplotlib.org/stable/tutorials/introductory/usage.html>`__, `Interactive Data Visualizations <https://jupyterbook.org/en/stable/interactive/interactive.html>`__
+    * Graphs, tables etc. are computed when the latest version of your project is built and published as a stand-alone website. All code examples on your website are validated each time you build.
+
+.. dropdown:: 📚 Bibliographies and external links
+
+    Maintain bibliography databases directly as code and have external links automatically verified.
+
+    Using extensions for Sphinx such as the popular `sphinxcontrib-bibtex <https://sphinxcontrib-bibtex.readthedocs.io/>`__ extension, you can maintain your bibliography with Sphinx directly or refer to entries ``.bib`` files, as well as generating entire Bibliography sections from those files.
+
+.. dropdown:: 📜 Modern themes and classic PDF outputs
+
+    .. figure:: /img/screenshot_rtd_downloads.png
+        :align: right
+
+    Use the latest state-of-the-art themes for web and have PDFs and e-book formats automatically generated.
+
+    New themes are improving every day, and when you write documentation based on Jupyter Book and Sphinx, you will separate your contents and semantics from your presentation logic. This way, you can keep up with the latest theme updates or try new themes.
+
+    Another example of the benefits from separating content and presentation logic: Your documentation also transforms into printable books and eBooks.
+
+.. dropdown:: 📐 Widgets, widgets and more widgets
+
+    Design your science project's layout and components with widgets from a rich eco-system of Open Source extensions built for many purposes. Special widgets help users display and interact with graphs, maps and more. :external+jupyterbook:doc:`Several <content/components>` `extensions <https://sphinx-gallery.github.io/>`__ are built and invented by the science community.
+
+.. dropdown:: ⚙️ Automatic builds
+
+    Build and publish your project for every change made through Git (GitHub, GitLab, BitBucket etc). Preview changes via Pull Requests. Receive notifications when something is wrong. How does this work? Have a look at this video:
+
+    .. TODO: Where should this video live?
+
+    .. video:: https://overtag.dk/files/enable-pull-request-builders.mp4
```

Comment:
Noted in chat, but this is better as an embed from a streaming service. A static embed can't be sampled down to a lower bitrate and takes more JS work to add features that an embed get us.

Reply:
With several options to choose from, I'm not sure which one is the right one to go with.

Using [sphinxcontrib-video](https://github.com/sphinx-contrib/video) and:

* Hosting our video tutorials as mp4 somewhere

Using [sphinxembeddedvideos](https://github.com/Peque/sphinxembeddedvideos) and:

* Hosting on archive.org (not sure which embed features it has)
* Hosting on a Peertube service (we need to choose or host one)
* Hosting on Youtube (tracking, ads etc)
* Paid Vimeo?